src/constants/categories.js

```diff
@@ -43,8 +43,8 @@ const categories:Array<Category> = [
         search: {
             q: "housing -(coordinating bodies) -(respite care) " +
                 "-(housing information) -hef",
-            service_type: "housing",
-            },
+                service_type: "housing",
```

Comment:
Almost... service_type was right, just the brace needed to move :)
You can put service_type back where it was